addons/external_intercom/functions/fnc_connect.sqf

```diff
@@ -23,6 +23,8 @@
 params ["_vehicle", "_player", "_actionParams"];
 _actionParams params [["_wireless", false, [false]]];
 
+TRACE_3('Connect',_player,_vehicle,_actionParams);
```

Comment:
But this looses where the thing came from, which is what the original trace did :sad:

Reply:
Well I don't have time to make it better so I shall just take what I got 😢